src/diffpy/pdffit2/__init__.py

```diff
@@ -17,15 +17,19 @@
 
 """PDFfit2 - real space structure refinement program."""
 
-# package version
+# Import package version to be used by C++ extensions
 from diffpy.pdffit2.output import redirect_stdout
-from diffpy.pdffit2.version import __version__, __date__
-from diffpy.pdffit2.pdffit import PdfFit
-from diffpy.pdffit2.pdffit2 import is_element
+from diffpy.pdffit2.version import __date__, __version__
```

Comment:
I finally found one origin of "import" problem. 

The C++ extension below `pdffit2.pdffit2` uses the `__verison__` global variable.

```
from diffpy.pdffit2.pdffit2 import is_element
```

But when we apply isort via pre-commit, the C++ module is imported first before `__version__` variable is initialized, causing the import issue.

Reply:
Good work @bobleesj .  I guess we need a fix? Is it to disable isort for this case? Or change how the imports are done?  It is a bit brittle if it can be broken by isort, but that fix is ok if we document it carefully so we don't break it again in the first when we try and do better linting....

Reply:
(thinking about a solution)